src/index.html

```diff
@@ -16,7 +16,11 @@
       href="./styles/index.scss"
```

Comment:
You should link to the compiled CSS file, not the SCSS source file. Make sure to compile your SCSS into CSS and link to the resulting CSS file in your HTML.